apps/nextcloud/templates/services/app/_env.yml.j2

```diff
@@ -0,0 +1,16 @@
+POSTGRES_DB: "{{ nextcloud_database_name }}"
+POSTGRES_HOST: "nextcloud-{{ nextcloud_database_host }}-{{ deployment_stamp }}"
+POSTGRES_PORT: "{{ nextcloud_database_port }}"
+NEXTCLOUD_ADMIN_USER: "{{ nextcloud_admin_user }}"
+NEXTCLOUD_CURRENT_DOMAIN: "{{ nextcloud_host }}"
+NEXTCLOUD_PREVIOUS_DOMAIN: "previous.{{ nextcloud_host }}"
+NEXTCLOUD_NEXT_DOMAIN: "next.{{ nextcloud_host }}"
+NEXTCLOUD_LOG_LEVEL: "{{ nextcloud_log_level }}"
+NEXTCLOUD_LOG_TYPE: "{{ nextcloud_log_type }}"
+NEXTCLOUD_REDIS_HOST: "{{ nextcloud_redis_host }}"
+NEXTCLOUD_REDIS_PORT: "{{ nextcloud_redis_port }}"
+NEXTCLOUD_DEBUG: "{{ nextcloud_debug }}"
+NEXTCLOUD_VERSION: "{{ nextcloud_version }}"
+USER_NAME: "{{ nextcloud_container_user }}"
+HOME: "{{ nextcloud_container_home_dir }}"
```

Comment:
Can we prefix those variable names?

Reply:
Yes but this is the variable we use in every docker project (richie, marsha)

Reply:
I changed with `USER_HOME`